index.html

```diff
@@ -56,17 +57,110 @@ <h2>Multi-Post Stories</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
+                </div>
+                <div class="card-2">
+                    <h2>Professional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
+                </div>
+                <div class="card-2">
+                    <h2>Professional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
+                </div>
+                <div class="card-2">
+                    <h2>Professional Art <br>Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <ul>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button class="project" type="button">See Project </button>
                 </div>
             </div> 
+        </section>
+    </div>
+    <div class="container-3">
+        <div class="About">
+            <h1>About Me</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <BUtton class="resume">Get My Resume</BUtton>
+            <img src="assets/Frame.png"  class= "write" alt="">
+            
+            
+            <section class="final">
+                <div class="list-1">
+                    <h3 class="lan">Languages</h3>
+                    <ul>
+                        <li>JavaScript</li>
+                        <li>Ruby</li>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                    </ul>
+                </div>
+                <div class="list-2">
+                    <h3 class="fram"> Framworks</h3>
```

Comment:
- [ ] As suggested by the previous code reviewer, I kindly suggest that you add icons for language, framework, and skills.
![image](https://user-images.githubusercontent.com/52934128/177207732-759da895-10e5-4de6-a3e3-8fa8480e07a9.png)